packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -266,25 +264,36 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
     override
     nonReentrant
   {
+    // called by aragon court
     require(_ruling <= DISPUTES_POSSIBLE_OUTCOMES, "invalid ruling");
     uint256 index = disputes[_disputeId];
     Locker storage locker = lockers[index];
     require(locker.resolverType == ADR.ARAGON_COURT, "!aragon");
     require(locker.locked, "!locked");
     require(locker.total > locker.released, "released");
     require(_msgSender() == locker.resolver, "!resolver");
-    require(_msgSender() != locker.client, "resolver == client");
 
     uint256 remainder = locker.total.sub(locker.released);
 
-    if (_ruling == DISPUTES_RULING_CLIENT) {
+    if (_ruling == DISPUTES_RULING_100_0) {
       IERC20(locker.token).safeTransfer(locker.client, remainder);
-    } else if (_ruling == DISPUTES_RULING_PROVIDERS) {
+    } else if (_ruling == DISPUTES_RULING_0_100) {
       IERC20(locker.token).safeTransfer(locker.provider, remainder);
+    } else if (_ruling == DISPUTES_RULING_75_25) {
+      uint256 clientAward = remainder.mul(3).div(4);
+      uint256 providerAward = remainder.sub(clientAward);
+      IERC20(locker.token).safeTransfer(locker.client, clientAward);
+      IERC20(locker.token).safeTransfer(locker.provider, providerAward);
+    } else if (_ruling == DISPUTES_RULING_25_75) {
+      uint256 clientAward = remainder.mul(1).div(4);
+      uint256 providerAward = remainder.sub(clientAward);
+      IERC20(locker.token).safeTransfer(locker.client, clientAward);
+      IERC20(locker.token).safeTransfer(locker.provider, providerAward);
     } else {
-      IERC20(locker.token).safeTransfer(locker.client, remainder / 2);
-      remainder = remainder - remainder / 2;
-      IERC20(locker.token).safeTransfer(locker.provider, remainder);
+      uint256 clientAward = remainder.div(2);
+      uint256 providerAward = remainder.sub(clientAward);
+      IERC20(locker.token).safeTransfer(locker.client, clientAward);
+      IERC20(locker.token).safeTransfer(locker.provider, providerAward);
```

Comment:
> lines 285 and 286: use safeMath div here
line 286: simpler to do remainder = remainder.div(2); ?

done. 
`remainder = remainder.sub(remainder.div(2))` may not be the same as  `remainder = remainder.div(2)` if remainder is odd.

Reply:
good point!